apps/pm-user/src/components/molecules/signUpForm/SignUpForm.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { Button, Form, Input, InputPassword, UserOutlined, MailOutlined, KeyOutlined } from '../../atoms';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+import styles from './signUpForm.module.scss';
+
+const SignUpForm = () => {
+  const history = useHistory();
+
+  async function signUpUser(firstName, lastName, email, password) {
+    try {
+      const response = await axios.post('http://localhost:3333/api/v1/newuser', {
+        profile: {
+          firstName,
+          lastName,
+          email,
+          login: email,
+        },
+        credentials: {
+          password: {
+            value: password,
+          },
+        },
+      });
+      console.log(response);
+      history.push('/recommendations');
+    } catch (err) {
+      console.log('error:', err);
+    }
+  }
+
+  const onFinish = (values) => {
+    if (values.confirmPassword === values.password) {
+      console.log('Success:', values);
+      signUpUser(values.firstName, values.lastName, values.email, values.password);
+    } else {
+      alert('password and confirmPassword should be same');
```

Comment:
Instead of alert, showing red text that the two fields are not matching is better.
Alerts are forceful way of conveying information.
Red text warnings.. are softer way in my opinion.

Reply:
yes we have to create an error component for it that is why written it here temporarily

Reply:
Ok. You could also show a simple notification if the passwords dont match.